### DISCUSSION TITLE
[BP#87] Utiliser une électricité issue d'EnR

### DISCUSSION THREAD

**Discussion by ACTLEM** [cnumr/best-practices#157] [Bonnes pratiques] [answered]
Par rapport à la discussion https://github.com/cnumr/best-practices/discussions/38 ("Hébergeur responsable"), je me demande s'il ne faut pas intégrer la [BP 87](https://github.com/cnumr/best-practices/blob/main/chapters/BP_087_fr.md) à la BP 86.

**Comment by devcrafting** [comment]
Cette BP peut être clivante (cf. Débat pro-nuke/pro-EnR...). Je ne sais pas trop quoi en penser...il faudrait peut-être expliciter pourquoi on fait le choix de mettre les EnR en avant : bilan carbone, déchets, consommation de matières premières, impacts systémiques pour gérer l'intermittence...

**Reply by ACTLEM** [reply]
Je suis d'accord sur le côté clivant, c'est d'ailleurs pour cela que dans la BP il est mentionné le côté Hydro électrique qui a l'avantage de moins faire débat.
Personnellement, je supprimerais cette BP sachant que la BP 86 existe et va être remaniée.

**Reply by Trankilloman** [reply]
Cette BP concerne également le fournisseur d'énergie de la structure qui fait le développement voir, si elle gère les périphériques des clients (par exemple si la structure met à disposition des pc, comme au pôle emploi ou lors d'une exposition), l'énergie de ceux ci. De fait, ce point là est plus large que les seuls datacenter. De fait, je pense qu'il serait préférable de la tourner de la sorte : utiliser des consommables et des sous traitants ayant une démarche ecoresponsable (car pour un site qui demanderait un lourd développement demandant beaucoup de réunions et une durée de vie très courte, il serait également judicieux de choisir des déplacements moins impactants).

**Reply by ACTLEM** [reply]
Oui, c'est vrai qu'il ne s'agit pas uniquement des DC ici. Tu as raison. Après, on est à la limite de la BP Numérique Responsable au sens large même si elle s'applique bien dans le cas que tu énonces: lourd développement et durée de vie courte.

**Reply by Trankilloman** [reply]
Dans ce cas, je suis d'accord avec la suppression de cette BP pour l'intégrer dans la BP #38

**Reply by DocRoms** [reply]
Je suis aussi d'avis de la supprimer et d'inclure les recommandations dans la BP#86 (cf #38)

**Comment by ACTLEM** [comment]
Concernant l'intégration à la BP86, il y a le problème que les certificats de garantie d’origine peuvent être acheter sans changer d'hébergeurs ou ne sont issus d'hébergeurs. 
Etant d'accord avec @devcrafting, l'idée serait peut-être de s'assurer que l'électricité fournie n'a pas été produite à partir de la combustion de ressources fossiles?

**Reply by tbroyer** [reply]
À propos de la garantie d'origine: https://twitter.com/maxcordiez/status/1457051276521377796

**Reply by ACTLEM** [reply]
Oui, toujours compliqué car des opérateurs profitent des failles mais aujourd'hui qu'a-t-on de mieux? J'avoue que je ne sais pas trop quoi faire de cette BP:
- La supprimer simplement
- La garder et mettre un disclaimer sur le fait qu'aujourd'hui c'est faute de mieux que de proposer des CGO (Frédéric a proposé de la réécrire en ce sens)

**Reply by tbroyer** [reply]
Quid de choisir un data center dans un pays dont l'énergie est massivement décarbonnée ? (e.g. en Europe préférer la France, la Suisse ou la Suède à l'Allemagne ou la Pologne) ou qui produit lui-même son électricité décarbonnée ?
Ce serait déjà une première étape qui devrait probablement faire consensus (même si les émissions de CO² ne sont pas le seul impact à prendre en compte).
On pourrait du coup faire un lien vers Electricity Map pour guider ce choix.

Mais on pourrait très bien fusionner cette BP avec la BP#86 et l'amender avec ces suggestions.

**Comment by ACTLEM** [comment, marked as answer]
Issue créée: https://github.com/cnumr/best-practices/issues/249

Disclaimer sur CGO (faute de mieux), lien sur Electricity Map, éviter le clivage ENR/Nucléaire.
